build/helper/metadata_add_all.py

```diff
@@ -337,6 +338,10 @@ def add_all_function_metadata(functions, config):
 
     for f in functions:
         _add_codegen_method(functions[f])
+        # NI-Digital doesn't have any documentation yet, so make sure the
```

Comment:
Is this meant to be permanent?
If so, I'd reword "Some drivers may not have documentation, so make sure the documentation key exists".
If not, then mark with TODO